libcxx/include/ccomplex

```diff
@@ -23,4 +23,17 @@
 #  pragma GCC system_header
 #endif
 
+#if _LIBCPP_STD_VER >= 17
+
+using __standard_header_ccomplex
+#  if _LIBCPP_STD_VER >= 20
+    _LIBCPP_DEPRECATED_("removed in C++20. Include <complex> instead.")
+#  else
+    _LIBCPP_DEPRECATED_("Include <complex> instead.")
+#  endif
+    = void;
+using __use_standard_header_ccomplex = __standard_header_ccomplex;
+
+#endif
```

Comment:
```suggestion
#  if _LIBCPP_STD_VER >= 20
using __standard_header_ccomplex _LIBCPP_DEPRECATED_("removed in C++20. Include <complex> instead.") = void;
  using __use_standard_header_ccomplex = __standard_header_ccomplex;
#  elif _LIBCPP_STD_VER >= 17
  using __standard_header_ccomplex _LIBCPP_DEPRECATED_("Include <complex> instead.") = void;
  using __use_standard_header_ccomplex = __standard_header_ccomplex;
#endif
```

Maybe 1-2 lines longer but easier to follow IMO.

The same comment applies to other headers.

Reply:
Done.